packages/module/playground/nuxt.config.ts

```diff
@@ -1,5 +1,5 @@
 export default defineNuxtConfig({
-  modules: ['@nuxtjs/tailwindcss', '../src/module'],
+  modules: ['@nuxtjs/tailwindcss', 'shadcn-nuxt'],
```

Comment:
```suggestion
  modules: ['@nuxtjs/tailwindcss', '../src/module'],
```

I think this might be better instead of linking module inside playground package.json like 
`"shadcn-nuxt": "file:.."`

Reply:
I've added it to `package.json` so that the deps is installed. We can do the change you suggested and keep the module entry in `package.json` but I think it would be confusing a bit. Also adding the module name in the config mimics the real usage of the module. What do you think?